rebuild_wrappers.py

```diff
@@ -815,8 +825,9 @@ def main(root: str, files: Iterable[Filename], ver: str):
 	gbl_tmp: JumbledGlobals
 	red_tmp: JumbledRedirects
 	fsp_tmp: JumbledFilesSpecific
+	alltypedefs: JumbledTypedefs
 	
-	gbl_tmp, red_tmp, fsp_tmp = readFiles(files)
+	gbl_tmp, red_tmp, fsp_tmp, alltypedefs = readFiles(files)
```

Comment:
Please also sort the typedef structure (in the `sortArrays` function, similar to the first element of the values of `fsp_tmp`), so that the order in which the content of `x64printer.c` is generated stays constant from one run to another.